src/styles.css

```
@@ -1,710 +1,270 @@
-  --ink:#1C1008;--ink2:#3A2010;--ink3:#7A5030;--muted:#B07850;
-  --border:#EAD8C8;--border2:#F2E5D8;
-  --green:#00875A;--red:#C53030;--blue:#1A56DB;--purple:#6B21A8;
-  --shadow:0 1px 4px rgba(28,16,8,.07),0 6px 20px rgba(28,16,8,.05);
-  --shadow-lg:0 12px 40px rgba(28,16,8,.13);
-  --card-bg:#FFFFFF;--nav-bg:rgba(250,250,250,.93);--toggle-bg:#EAD8C8;--toggle-knob:#FFFFFF;
-  --input-bg:#FAFAFA;--tag-bg:#FAFAFA;--modal-bg:#FFFFFF;--an-bg:#FAFAFA;
-  --skeleton-base:#e8e8e8;--skeleton-shine:#f4f4f4;
+  --bg:#FAFAFA;
+  --white:#FFFFFF;
+
+  --orange:#F46B0E;
```

Stylelint (17.11.1)

[error] Expected no empty line before custom property (custom-property-empty-line-before)

(custom-property-empty-line-before)